tests/unit/objects/test_groups.py

```diff
@@ -108,6 +113,13 @@ def resp_groups():
             content_type="application/json",
             status=200,
         )
+        rsps.add(
+            method=responses.POST,
+            url="http://localhost/api/v4/groups/1/service_accounts",
+            json=service_account_content,
+            content_type="application/json",
+            status=200,
+        )
```

Comment:
I don't think we need this one here - it should be fine to use the `resp_create_group_service_account` fixture you have below.